apps/learner/src/routes/(protected)/+layout.svelte

```diff
@@ -50,6 +66,18 @@
   </div>
 </div>
 
-<NowPlayingView isopen={isNowPlayingViewOpen} onclose={handleNowPlayingViewClose} />
-
+<NowPlayingView
+  isopen={isNowPlayingViewOpen}
+  onclose={handleNowPlayingViewClose}
+  isplaying={player.isPlaying}
+  playbackspeed={player.playbackSpeed}
+  duration={player.duration}
+  progress={player.progress}
+  currenttrack={player.currentTrack}
```

Comment:
Add a guard to make sure NowPlayingView can only be rendered when there is a currenttrack.